openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -523,6 +523,7 @@ def post(self, request, *args, **kwargs):
     def dispatch(self, *args, **kwargs):
         self.uidb36 = kwargs.get('uidb36')
         self.token = kwargs.get('token')
+        self.isNewUser = kwargs.get('method', None) == 'new_password'
```

Comment:
In pyhton we use snake case naming convention
isNewUser -> is_new_user

Reply:
updated